### ISSUE TITLE
Multiline String Parameter Turkish Character Problem

### ISSUE THREAD

**Issue by urunsal** [MCMicS/jenkins-control-plugin#270]
We have a Turkish character problem in Multiline Strin Parameter. Could you please help me?

For example, the letter I or the letter S is written as a question mark instead.
Thanks.

**Comment by urunsal** [comment]
example : İ Ö Ç Ş Ğ  -> ? ?

**Comment by MCMicS** [comment]
Thank for report. I look at this next week after easter holiday. I have an idea what the problem is.

I try to fix this then. I currently plan to push a new version (eap) next week with some change. Maybe this is fixed

**Comment by urunsal** [comment]
First of all, thank you for your interest. Instead of any turkish character in a paragraph? is printing the mark. For example instead of ISTANBUL like? STANBUL
TURKEY place or T? RK? YE. I would appreciate it if you could help me. Have a nice day.

**Comment by urunsal** [comment]
First of all, thank you for your interest. Instead of any turkish character in a paragraph? is printing the mark. For example instead of ISTANBUL like? STANBUL
TURKEY place or T? RK? YE. I would appreciate it if you could help me. Have a nice day.

**Comment by MCMicS** [comment]
Thank for clarification.
i will look at this monday or thuesday

**Comment by MCMicS** [comment]
@urunsal Sorry for the the question. But i not know the turkish alphabet :)

> ISTANBUL like? STANBUL
I've tried this and it works in my environment.
Is this a normal 'I' or the turkish letter (İ)
`İstanbul`

Right??
COuld you please provide the turkish written word of this so I can test it

![grafik](https://user-images.githubusercontent.com/4604206/113759030-e8742d00-9714-11eb-83ae-5469c0818fa3.png)
![grafik](https://user-images.githubusercontent.com/4604206/113759050-ef9b3b00-9714-11eb-9151-f204bed2b871.png)

**Comment by MCMicS** [comment]
I test following simple pieline:
```groovy
pipeline {
    agent any
    
    parameters {
        choice choices: ['ISTANBUL', 'İstanbul', 'İ Ö Ç Ş Ğ',  'TURKEY'], name: 'Choice'
        string 'Text'
        text description: '''İ Ö Ç Ş Ğ İstanbul'', name: 'Multiline'
    }

    stages {
        stage('Params') {
            steps {
                echo "Choice: $params.Choice"
                echo "Text: $params.Text"
                echo "Multiline: $params.Multiline"
            }
        }
    }
}

```

**Comment by MCMicS** [comment]
Colud you please try following EAP Build?
https://plugins.jetbrains.com/plugin/6110-jenkins-control/versions/eap/115959

I will test it more this week

**Comment by MCMicS** [comment]
released with [0.13.11](../releases/tag/v0.13.11)
